processing/src/main/java/org/apache/carbondata/processing/loading/steps/InputProcessorStepWithNoConverterImpl.java

```diff
@@ -63,6 +64,8 @@
 
   private Map<Integer, GenericDataType> dataFieldsWithComplexDataType;
 
+  private RowConverterImpl rowConverter;
```

Comment:
Why InputProcessorStepWithNoConverterImpl need RowConverter its better to call direct interface of index generator because for other columns no need to convert the data

Reply:
Have skipped the converter step for schema columns. And did the index column value generation and field conversion through FieldConverter for index column(IndexFieldConverterImpl).